server/app/durablejobs/jobs/FixApplicantDobDataPathJob.java

```diff
@@ -0,0 +1,56 @@
+package durablejobs.jobs;
+
+import com.google.common.base.Preconditions;
+import durablejobs.DurableJob;
+import io.ebean.DB;
+import io.ebean.Database;
+import java.time.LocalDate;
+import models.Applicant;
+import models.PersistedDurableJob;
+import services.WellKnownPaths;
+import services.applicant.ApplicantData;
+
+/**
+ * Finds applicant data objects with applicant_date_of_birth set to a number and changes it to a
+ * nest object.
+ */
```

Comment:
suggestion (blocking): please add the "why" in addition to the "what"

Reply:
done